assemblies/features/src/main/resources/features.xml

```diff
@@ -55,7 +55,7 @@
     <bundle>mvn:pentaho/pentaho-hadoop-shims-mapr-osgi-jaas/${project.version}</bundle>
   </feature>
   <feature name="pentaho-big-data-kettle-plugins" version="1.0">
-    <feature>pentaho-big-data-plugin-api</feature>
+    <feature>pentaho-big-data-plugin-shim-impl</feature>
```

Comment:
Hudak, the following bundle needed "com.pentaho.big.data.bundles.impl.shim.common" which is supplied by the impl feature today

      <bundle>mvn:pentaho/pentaho-big-data-kettle-plugins-common-named-cluster-bridge/${project.version}